README.md

```diff
@@ -2,6 +2,9 @@
 
 Liberate MARC data from Voyager.
 
+[![CircleCI](https://circleci.com/gh/pulibrary/marc_liberation.svg?style=svg)](https://circleci.com/gh/pulibrary/marc_liberation)
+[![CoverageStatus](https://coveralls.io/repos/github/pulibrary/marc_liberation/badge.svg?branch=main)](https://coveralls.io/github/pulibrary/marc_liberation?branch=main)
+[![InchCI](https://inch-ci.org/github/pulibrary/marc_liberation.svg?branch=main)](https://inch-ci.org/github/pulibrary/marc_liberation)
 ## Services
```

Comment:
I can't see the docs badge after changing master to main.

Reply:
There may be a delay on the servers for inch-ci.org.

Reply:
when I https://inch-ci.org/github/pulibrary/marc_liberation?pending_build=607563 it never builds. maybe I need to configure something more?

Reply:
I meant to write when I try to build the branch name: https://inch-ci.org/github/pulibrary/marc_liberation?branch=693-yard&pending_build=607564

Reply:
it is interesting that I can see https://inch-ci.org/github/pulibrary/marc_liberation?branch=master but not  https://inch-ci.org/github/pulibrary/marc_liberation?branch=main.